Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/model/EntityMeadowlarkData.ts

```diff
@@ -40,12 +40,17 @@ export function addEntityMeadowlarkDataTo(entity: ModelBase) {
  * Initialize all properties with Meadowlark data placeholder.
  */
 export function enhance(metaEd: MetaEdEnvironment): EnhancerResult {
-  getAllEntitiesOfType(metaEd, 'domainEntity', 'association', 'domainEntitySubclass', 'associationSubclass').forEach(
-    (entity) => {
-      if (entity.data.meadowlark == null) entity.data.meadowlark = {};
-      addEntityMeadowlarkDataTo(entity);
-    },
-  );
+  getAllEntitiesOfType(
+    metaEd,
+    'domainEntity',
+    'association',
+    'domainEntitySubclass',
+    'associationSubclass',
+    'descriptor',
+  ).forEach((entity) => {
```

Comment:
This is an initializer for the Meadowlark entity metadata object. Now that descriptors have metadata, they need to be initialized with it. Prettier made this look bigger than it is.